packages/common/src/hooks/purchaseContent/constants.ts

```diff
@@ -5,3 +5,5 @@ export const PURCHASE_METHOD = 'purchaseMethod'
 // Pay between $1 and $100 extra
 export const minimumPayExtraAmountCents = 100
 export const maximumPayExtraAmountCents = 10000
+
+export const CENTS_TO_USDC_MULTIPLIER = 10000
```

Comment:
Oh haha, I should have been more specific. I was thinking the DIVISOR stuff that @rickyrombo added. But this is fine for now!

Reply:
oh gotcha, yeah i don't think we have one for 10000 there yet.